integration/tests/basic_functionality/negative/neg_deploy_with_missing_r10k_yaml.rb

```diff
@@ -9,7 +9,7 @@
 r10k_fqp = get_r10k_fqp(master)
 
 #Verification
-error_message_regex = /No configuration file given, no config file found in current directory, and no global config present/
+error_message_regex = /the supplied configuration does not define the 'sources' key/
```

Comment:
The config file loading happens far before we start picking out values from the config, and supplying a config is no longer a hard requirement for running r10k. It's possible to plumb that through but it'll actually be a bit of a pain.